src/common/utils/modulesRegistry.h

```diff
@@ -15,6 +15,17 @@ namespace NonLocalizable
     const static wchar_t* ExtensionsID = L"extensions";
     const static wchar_t* MDExtension = L".md";
     const static wchar_t* SVGExtension = L".svg";
+    const static std::vector<std::wstring> ExtSVG      = { L".svg" };
+    const static std::vector<std::wstring> ExtMarkdown = { L".md", L".markdown", L".mdown", L".mkdn", L".mkd", L".mdwn", L".mdtxt", L".mdtext" };
+    const static std::vector<std::wstring> ExtPDF      = { L".pdf" };
+    const static std::vector<std::wstring> ExtGCode    = { L".gcode" };
+    const static std::vector<std::wstring> ExtSTL      = { L".stl" };
+    const static std::vector<std::wstring> ExtNoNoNo   = { 
```

Comment:
Let's go without this for now. Although, I understand your frustration, it is subjective. :)

e.g. I personally like .txt preview as it themed and it doesn't break lines (which makes it unreadable IMHO), but gives the horizontal scroll.

Reply:
Huh? What am I looking at? Did you intend to submit a preview on lines 24 through 27? (What I am seeing here is lines 20 through 23.)

Reply:
I have an open PR which enables you switching text wrapping.

But what do you mean by buggy Ini and reg files. Could you send a screenshot?

Reply:
Screenshots? Sure. 👍

In the case of plain text files, Windows' native preview handler is orders of magnitude better. Here is a shot of the Monaco preview handler, which renders most of the text as garbled and unreadable:

![TXT Bad render (Monaco fallback renderer)](https://user-images.githubusercontent.com/17097175/155137555-173d3682-006f-4899-b710-8a91ec23f1a1.png)

Now, Windows' native handler:

![TXT Good render (Windows native)](https://user-images.githubusercontent.com/17097175/155137597-22fe17ef-58b0-4614-a38a-97dcdb4360d6.png)

As for INI files, there is a second problem, in addition to the one you see in the plain text files above. Monaco syntax highlighter colorizes parts of the text without regards to its meaning:

![INI Bad render (Monaco fallback renderer)](https://user-images.githubusercontent.com/17097175/155137820-fce33030-9cec-4f71-86a6-57850b79c117.png)

Windows' native handler is fine:
![INI Good render (Windows native)](https://user-images.githubusercontent.com/17097175/155137866-6c60d88a-39e8-487b-954e-b5c2a5b663d9.png)

Reply:
sorry, I meant without `ExtNoNoNo`

Reply:
> In the case of plain text files, Windows' native preview handler is orders of magnitude better. Here is a shot of the Monaco preview handler, which renders most of the text as garbled and unreadable:

This is also fixed, and will be included in next PT release

Reply:
I will investigate in the problem with the ini files and most likely open a issue in the monaco editor repo

Reply:
> This is also fixed, and will be included in next PT release
That's good news, but are you sure? They told me they'd never fix stuff like this because they aren't relevant to developers, and fixing them is too complicated. I'd appreciate a link to the fixing PR or commit.

Reply:
https://github.com/microsoft/PowerToys/pull/16108

Reply:
> > This is also fixed, and will be included in next PT release
> 
> That's good news, but are you sure? They told me they'd never fix stuff like this because they aren't relevant to developers, and fixing them is too complicated. I'd appreciate a link to the fixing PR or commit.

Who told you this?

Reply:
Thank you for your patience, gentlemen. I've compiled and tested the main branch containing #16108. It indeed handles plain text files well:

![TXT Good render (Monaco fallback handler)](https://user-images.githubusercontent.com/17097175/155196352-4e580f41-bfa0-4080-af45-28686b92479b.png)

Hence, I am more than happy to oblige you and remove the exclusions. 

However, I have added a new exclusion: .SVGZ. The Monaco preview handler cannot handle this file type:

![SVGZ Fail render (Monaco fallback handler)](https://user-images.githubusercontent.com/17097175/155196586-c2099555-1cec-47d2-8fe5-2a899b403990.png)

Can I mark this as resolved?

@Aaron-Junker 

> Who told you this?

Someone in a Microsoft AMA. I don't remember who. There was no point in memorizing her name.

Reply:
> However, I have added a new exclusion: .SVGZ. The Monaco preview handler cannot handle this file type:

This is known issue https://github.com/microsoft/PowerToys/issues/16047. However, I think SVG handler also doesn't handle svgz correctly. So, fixing this is a different issue, no need to dig deep into it here